### ISSUE TITLE
Some questions in dice loss

### ISSUE THREAD

**Issue by lingzerocy** [Hsuxu/Loss_ToolBox-PyTorch#11]
dice_loss.py
In line 102, `output = F.softmax(output, dim=1)`
in line 105, `dice_loss = dice(output[:, i], target[:, i])`
in line 51, `output = torch.sigmoid(output)`
softmax-->sigmoid？

**Comment by Hsuxu** [comment]
Thanks for your comments.  It wills cause apply twice activation function. I added an option in function `forward` in `BinaryDiceLoss` https://github.com/Hsuxu/Loss_ToolBox-PyTorch/blob/f7a56cd92bf226117080a9c6a6342c17aa30d7e2/DiceLoss/dice_loss.py#L49
